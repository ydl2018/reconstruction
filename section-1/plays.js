export default {
    "hamlet":{"name":"hamlet","type":"tragedy"},
    "as-like":{"name":"As You Like it","type":"comedy"},
    "othello":{"name":"othello","type":"tragedy"}
}