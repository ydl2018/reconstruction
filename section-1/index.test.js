import statement from "./index";
import plays from './plays.js'

import invoices from './invoices.js';

test('print acoount list',()=>{
    expect(statement(invoices,plays)).toMatchSnapshot()
})
