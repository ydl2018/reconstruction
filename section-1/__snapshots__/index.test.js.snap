// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print acoount list 1`] = `
"Statement for BigCo
hamlet: $290.00 55  seats
As You Like it: $155.00 35  seats
othello: $140.00 40  seats
amount owned is $290.00
you earn 47 credits 
"
`;
